src/crypto/crypto_tls.cc

```diff
@@ -1492,8 +1468,9 @@ void TLSWrap::MemoryInfo(MemoryTracker* tracker) const {
   tracker->TrackField("sni_context", sni_context_);
   tracker->TrackField("error", error_);
   tracker->TrackFieldWithSize("pending_cleartext_input",
-                              pending_cleartext_input_.size(),
-                              "AllocatedBuffer");
+                              pending_cleartext_input_ ?
+                                  pending_cleartext_input_->ByteLength() : 0,
+                              "std::unique_ptr<v8::BackingStore>");
```

Comment:
We can just use `TrackField()` for BackingStores, I think.

Reply:
Sure, done!